docset/windows/smbshare/Set-SmbServerConfiguration.md

```diff
@@ -50,46 +50,46 @@ The **Set-SmbServerConfiguration** cmdlet sets the Server Message Block (SMB) Se
 ## EXAMPLES
 
 ### Example 1: Set the SMB Service configuration
-```
+```powershell
 PS C:\>Set-SmbServerConfiguration -MaxChannelPerSession 16
-Confirm 
-Are you sure you want to perform this action? 
-Performing operation 'Modify' on Target 'SMB Service Configuration'. 
+Confirm
```

Comment:
Can you please pull out the confirm, etc. dialog? This typically isn't part of PoSh cmdlet examples.

Reply:
Isn't the purpose of this example to show that it usually asks for confirmation?